leo/commands/setup.rs

```diff
@@ -14,82 +14,75 @@
 // You should have received a copy of the GNU General Public License
 // along with the Leo library. If not, see <https://www.gnu.org/licenses/>.
 
-use crate::{
-    cli::*,
-    cli_types::*,
-    commands::BuildCommand,
-    errors::{CLIError, RunError},
-};
+use crate::{commands::Command, context::Context};
+use anyhow::{anyhow, Result};
+
 use leo_compiler::{compiler::Compiler, group::targets::edwards_bls12::EdwardsGroupType};
 use leo_package::{
     outputs::{ProvingKeyFile, VerificationKeyFile},
-    root::Manifest,
     source::{MAIN_FILENAME, SOURCE_DIRECTORY_NAME},
 };
 
+use rand::thread_rng;
 use snarkvm_algorithms::snark::groth16::{Groth16, Parameters, PreparedVerifyingKey, VerifyingKey};
 use snarkvm_curves::bls12_377::{Bls12_377, Fr};
 use snarkvm_models::algorithms::snark::SNARK;
 
-use clap::ArgMatches;
-use rand::thread_rng;
-use std::{convert::TryFrom, env::current_dir, time::Instant};
+use structopt::StructOpt;
+
+use super::build::Build;
+use tracing::span::Span;
 
-#[derive(Debug)]
-pub struct SetupCommand;
+/// Run setup ceremony for Leo program Command
```

Comment:
```suggestion
/// Executes the setup command for a Leo program
```

Reply:
Doc comments are command descriptions with StructOpt. Fixed this to "Run a program setup" as in current leo binary.